vendor-extra/ContentPageBundle/src/Controller/ContentController.php

```diff
@@ -48,17 +49,20 @@ public function __invoke(Request $request, string $id) : Response
             )
             ->then(
                 function (ResponseInterface $response) use ($request) : Response {
-                    $dom = FluentDOM::load((string) $response->getBody());
+                    $document = FluentDOM::load((string) $response->getBody());
 
                     $context = [
                         'lang' => $request->getLocale(),
                         'dir' => text_direction($request->getLocale()),
                     ];
 
+                    $event = new CreateContentPageEvent($document, $context);
```

Comment:
naming matches the example https://github.com/symfony/symfony/tree/4.2/src/Symfony/Component/HttpKernel/Event